test/functional/apps/dashboard/embeddable_rendering.js

```diff
@@ -92,7 +92,7 @@ export default function ({ getService, getPageObjects }) {
   };
 
   // FLAKY: https://github.com/elastic/kibana/issues/33504
```

Comment:
Remove this line? Or maybe move down to where the geo map is being filtered out.

Reply:
Done! Removed it for now. Thinking the `TODO` should be enough to point us in the right direction for future work. Happy to change it if you have any strong preferences though!

Reply:
nope, sgtm!